content/applications/finance/accounting/fiscal_localizations/localizations/kenya.rst

```diff
@@ -0,0 +1,119 @@
+=====
+Kenya
+=====
+
+.. _localization/kenya/configuration:
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the following modules to get all the features of the Kenyan
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Kenyan - Accounting`
+     - `l10n_ke`
+     - Installing this module grants you access to the list of accounts used in the local GAAP and
+       the list of common taxes (VAT, etc.).
+   * - :guilabel:`Kenyan - Accounting Reports`
+     - `l10n_ke_reports`
+     - Installing this module grants you access to improved accounting reports for Kenya, such as
+       Profit and Loss and Balance Sheets.
+
+You also have to install the **Kenya Tremol Device EDI Integration** package to be able to report
+your taxes to the **Kenya Revenue Authority (KRA)** using the Tremol G03 Control Unit:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Kenya Tremol Device EDI Integration`
+     - `l10n_ke_edi_tremol`
+     - This module integrates with the Kenyan G03 Tremol control unit device to report taxes to KRA
+       through TIMS.
+
+.. image:: kenya/modules.png
+   :align: center
+   :alt: The three modules for the Kenya Fiscal Localization Package on Odoo
+
+Kenyan TIMS integration
+=======================
+
+The Kenya Revenue Authority (KRA) has decided to go digital for tax collection through the **Tax
+Invoice Management System (TIMS)**. As of December 1st, 2022, all VAT-registered persons should
+comply with TIMS. The goal is to reduce VAT fraud, increase tax revenue, and increase VAT compliance
+through standardization, validation, and transmission of invoices to KRA on a real-time or near
+real-time basis.
+
+All VAT-registered taxpayers should use a **compliant tax register**. Odoo decided to develop the
+integration of the **Tremol G03 Control Unit (type C)**, which can be run locally through USB. This
+device validates invoices to ensure financial documents meet the new regulations and send the
+validated tax invoices directly to KRA.
+
+Installing the Proxy Server on a Windows device
+-----------------------------------------------
+
+Go to odoo.com/download, fill out the required information and click :guilabel:`Download`.
+
+.. image:: kenya/download.png
+   :align: center
+   :alt: Install the Proxy Server on a Windows device
+
+Once it is loaded on your computer, a wizard opens. You have to read and agree with the terms of the
+agreement. On the next page, select the **type of install: Odoo IoT**. And then click
+:guilabel:`Next` and :guilabel:`Install`. Once completed, click :guilabel:`Next`. An **access token
+number** for the Odoo IoT appears. Copy it for later and click :guilabel:`Next`. Check the
+:guilabel:`Start Odoo` box to be redirected to Odoo automatically, and then click
+:guilabel:`Finish`.
```

Comment:
Why is it important to copy the IoT box token code? What would be its purpose?
I'd add a link to the "connect an IoT" doc because it's a bit "complex" - also, the IoT cannot be up and running if you don't link it to your database - maybe installing and selecting Odoo IOT as type does the trick but I couldn't test it  :/

Reply:
Josse confirmed it is not necessary to save the token. Text Updated.
See also to "connect an IoT" doc added.